website/docs/reference/node-selection/methods.md

```diff
@@ -210,6 +210,7 @@ Because state comparison is complex, and everyone's project is different, dbt su
 - `state:modified.relation`: Changes to `database`/`schema`/`alias` (the database representation of this node), irrespective of `target` values or `generate_x_name` macros
 - `state:modified.persisted_descriptions`: Changes to relation- or column-level `description`, _if and only if_ `persist_docs` is enabled at each level
 - `state:modified.macros`: Changes to upstream macros (whether called directly or indirectly by another macro)
+- `state:modified.vars`: Changes to [`vars`](/docs/build/project-variables) that a node depends on (available in v1.9 or with [dbt Cloud Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) via the `state_modified_compare_vars` [behavior flag](/reference/global-configs/behavior-changes))
```

Comment:
Should we specifically call out the need to set the flag? Otherwise looks 👍🏻 
```suggestion
- `state:modified.vars`: Changes to [`vars`](/docs/build/project-variables) that a node depends on (available in v1.9 or with [dbt Cloud Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) when you set the `state_modified_compare_vars` [behavior flag](/reference/global-configs/behavior-changes) to `true`)
```

Reply:
@runleonarun Open if you disagree, but I'm thinking we _should_ link to the related behavior flag. Otherwise, if someone only comes to this page, they wouldn't know that they need to opt-in to the behavior flag to get this functionality.

Reply:
100% agree! Do you think it's still ok to add the extra info "when you set...to `True`"?

Reply:
Yeah! Applied your suggested changes.